doc/api/crypto.md

```diff
@@ -3895,6 +3895,130 @@ Type: {Crypto} An implementation of the Web Crypto API standard.
 
 See the [Web Crypto API documentation][] for details.
 
+## `crypto` Promises API
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+The `crypto.promises` API provides an alternative set of asynchronous crypto
+methods that return `Promise` objects and execute operations in libuv's
+threadpool.
+The API is accessible via `require('crypto').promises` or `require('crypto/promises')`.
+
+### `cryptoPromises.diffieHellman(options)`
```

Comment:
Given that it's just the two, I would perhaps make the signature just `cryptoPromises.diffieHellman(privateKey, publicKey)`. It still leaves room for an options argument later if we needed.

Reply:
This is in sync with `crypto.diffieHellman` and is the format [requested then](https://github.com/nodejs/node/pull/31178#discussion_r362884242). For the sake of consistency, i'll keep it.